api/routes/router.go

```diff
@@ -26,6 +26,7 @@ func SetupRoutes(engine *gin.Engine) {
 
 	// Loomies
 	engine.POST("/loomies/near", middlewares.MustProvideAccessToken(), controllers.HandleNearLoomies)
+	engine.POST("/loomies/exists", middlewares.MustProvideAccessToken(), controllers.HandleValidateLoomieExists)
```

Comment:
What about receiving the Loomie Id as an endpoint parameter. Eg. `/loomies/exists/:id`. At this way the endpoint can use the `GET` method and can work without sending any JSON payload. Let me know if you'll implement this.

Reply:
Also consider what @Woynert said (Returning the Loomie ID).